docs/src/main/asciidoc/redis-support.adoc

```diff
@@ -29,7 +29,7 @@ NOTE: If you choose to use a security principal to authenticate and authorize wi
 [cols="4*", options="header"]
 |===
 |Property |Description |Default Value | Required
-|*spring.cloud.azure.redis*.enabled |Azure Cache for Redis instance name.|true | No
+|*spring.cloud.azure.redis*.enabled |A value that indicates whether the Azure Cache for Redis is enabled.|true | No
```

Comment:
To make sure MS docs and GitHub pages synced, please follow this principal:

1. Make MS docs page synced with GitHub pages in 4.0.0 tag.
2. Create draft PR to update MS docs for 4.1.0 version. Merge this PR after 4.1.0 released.
3. Copy diffs from MS docs to GitHub pages.